src/transport.ts

```diff
@@ -24,4 +37,21 @@ export class WebRTCTransport implements Transport {
 	get [symbol](): true {
 		return true
 	}
+
+	todo_cb() {
+	}
+
+	_connect (ma: Multiaddr, options: WebRTCDialOptions) {
+		this.channel = this.peer_connection.createDataChannel("data");
```

Comment:
`this.peer_connection` would be incorrect. A peerconnection should be scoped to a connection and not a transport.